gangplank/ocp/bc.go

```diff
@@ -208,6 +208,25 @@ binary build interface.`)
 		return err
 	}
 
+	// Get the latest build that might have happened
+	lastBuild, lastPath, err := cosa.ReadBuild(cosaSrvDir, "", cosa.BuilderArch())
+	if err == nil {
+		keyPath := filepath.Join(lastBuild.BuildID, cosa.BuilderArch())
+		l := log.WithFields(log.Fields{
+			"prior build": lastBuild.BuildID,
+			"path":        lastBuild,
+			"to path":     filepath.Join("srv", "bulds", keyPath),
```

Comment:
```suggestion
			"to path":     filepath.Join("srv", "builds", keyPath),
```

Reply:
Typos! My one weakness!


Good catch @miabbott

Reply:
My salvation is that its a log line :)